modules/ROOT/pages/deploy-partner-manager.adoc

```diff
@@ -0,0 +1,45 @@
+= Deploying Partner Manager Runtime Apps
+
+You can deploy Anypoint Partner Manager runtime apps to either CloudHub or Anypoint Runtime Fabric. 
+
+To configure Partner Manager deployment settings, use the *Partner Manager Settings* page in Access Management. You can configure separate deployment settings for each business group environment. You must have the `Partner Manager Administrator` permission to configure or modify deployment settings.
```

Comment:
Should this be access management per product tiering?

Reply:
I will change it, but Sara has Access Management throughout her doc.